build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Unfortunately, 2.2.3 breaks the unit tests. So we stick to 2.1.3 for now, until someone (probably me) puts in some effort to fix this.